src/pages/donghyeok/Login/Login.js

```diff
@@ -1,8 +1,101 @@
 import React from 'react';
+// import { Link } from 'react-router-dom';
+import './Login.scss';
 
 class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      id: '',
+      pw: '',
+      buttonColor: false,
+    };
+  }
+
+  handleIdInput = e => {
+    this.setState({
+      id: e.target.value,
+    });
+  };
+
+  handlePwInput = e => {
+    this.setState({
+      pw: e.target.value,
+    });
+  };
```

Comment:
계산된 속성명을 이용해 더 간단하게 작성할 수 있습니다!
단, input 태그의 name attribute에 해당하는 관련 state를 부여해주셔야 합니다.
```suggestion
  handleInput = e => {
    this.setState({
      [e.target.name]: e.target.value,
    });
  };
```